catalog/registry/catalog-registry-identificationplugin/src/test/java/ddf/catalog/registry/identification/IdentificationPluginTest.java

```diff
@@ -169,6 +184,52 @@ public void testIdsAlreadySet() throws Exception {
 
     }
 
+    @Test
+    public void testRemoteRequest() throws Exception {
+        String xml = convert("/registry-no-extid.xml");
+        sampleData.setAttribute(Metacard.METADATA, xml);
+        Map<String, Serializable> props = new HashMap<>();
+        props.put(Constants.LOCAL_DESTINATION_KEY, false);
+        CreateRequest result = idp.process(new CreateRequestImpl(Collections.singletonList(
+                sampleData), props));
+        assertThat(result.getMetacards()
+                .get(0)
+                .getMetadata(), equalTo(xml));
+    }
+
+    @Test(expected = StopProcessingException.class)
+    public void testDuplicatChecking() throws Exception {
+        String xml = convert("/registry-both-extid.xml");
+        sampleData.setAttribute(Metacard.METADATA, xml);
+        idp.process(new CreateResponseImpl(new CreateRequestImpl(sampleData),
+                null,
+                Collections.singletonList(sampleData)));
+        idp.process(new CreateRequestImpl(sampleData));
+    }
+
+    @Test
+    public void testDuplicatCheckingAfterDelete() throws Exception {
+        String xml = convert("/registry-both-extid.xml");
+        sampleData.setAttribute(Metacard.METADATA, xml);
+        idp.process(new CreateResponseImpl(new CreateRequestImpl(sampleData),
+                null,
+                Collections.singletonList(sampleData)));
+        idp.process(new DeleteResponseImpl(new DeleteRequestImpl(Collections.singletonList("abc123"),
+                RegistryObjectMetacardType.REGISTRY_ID,
+                null), null, Collections.singletonList(sampleData)));
+        idp.process(new CreateRequestImpl(sampleData));
+    }
+
+    @Test
+    public void testUnusedMethods() throws Exception {
```

Comment:
I don't really have any strong opinions about this, but I don't feel like having these kinds of methods adds any value to the code.